Source/WebInspectorUI/UserInterface/Views/Popover.css

```diff
@@ -34,8 +34,13 @@
     --popover-background-color: var(--panel-background-color);
     --popover-border-color: hsla(0, 0%, 0%, 0.25);
     --popover-shadow-color: hsla(0, 0%, 0%, 0.5);
+}
 
-    background-image: -webkit-canvas(popover);
+.popover.background-canvas {
```

Comment:
Do we really want to apply all the styles from `.popover` on the canvas? Should it be `.popover > .background-canvas` rather?